packet/checksum_test.go

```diff
@@ -2,18 +2,17 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-package packet_test
+package packet
```

Comment:
It is not very critical, I am just not sure we need make all tests internal.
In this file we have tests for only API functions, it should be ok to leave it packet_test as assumed for API tests.

Reply:
We don't want to have multiple packages